coretex/cli/config.py

```diff
@@ -0,0 +1,193 @@
+from pathlib import Path
+
+import os
+
+from tabulate import tabulate
+
+import click
+
+from ..configuration import loadConfig, saveConfig, isUserConfigured, isNodeConfigured
+from ..networking.network_manager import NetworkManager
+
+
+SERVER_URL = "dev.biomechservices.com"
+NODE_PORT = 5443
+CORETEX_CONFIG_DIR = Path.home() / ".config" / "coretex"
+
+
+def configUser() -> None:
+    config = loadConfig()
+    if isUserConfigured(config):
+        headers = ["username", "server", "storage path"]
+
+        click.echo(click.style("Current Configuration:", fg = "blue"))
+        click.echo(tabulate([[
+            config.get("username"),
+            config.get("serverUrl"),
+            config.get("storagePath")
+        ]], headers = headers))
+
+        if not click.prompt(
+            "User configuration already exists. Would you like to update (Y/n)?",
+            type = bool,
+            default = True,
+            show_default = False
+        ):
+            return
+
+    click.echo("Configuring user...")
+    while True:
```

Comment:
Limit retires for authenticate to 3 times after which an error message will be printed out